### PR TITLE
Introduced wrapping for stringprocessors

### PR DESCRIPTION
**Pull request by jcomack**
This will allow us to keep string processors separate from researches / the researcher and will allow them to remain generic.

### CODE REVIEW

**Comment by andizer** [comment]
CR done. Please fix the travis errors.

File: js/stringProcessing/countWords.js [hunk 1]
```diff
@@ -6,13 +6,11 @@ var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
...
- * @param {Paper} paper The paper to use for the word count.
+ * @param {string} text The text to be counted.
  * @returns {int} The word count of the given text.
  */
-module.exports = function( paper ) {
-
-	var text = stripTags( paper.getText() );
-	text = stripSpaces( text );
+module.exports = function( text ) {
+	var text = stripSpaces( stripTags( text ) );
```
Comment (andizer) [inline js/stringProcessing/countWords.js]: `var` isn't necessary here because you are overwriting the existing `text`

**Comment by terw-dan** [comment]
CR Done :+1:

**Comment by terw-dan** [comment]
Acceptance
somewhere we pass a paper in stead of a string, that causes the stripHTMLtags function to break

image

**Comment by atimmer** [comment]
CR3 Done 👍🏻

**Comment by atimmer** [comment]
Acceptance done 👍🏻

**Pull request merged**